index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="	https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav id="navbar" class="navbar navbar-expand navbar-dark">
+      <div class="container-fluid">
+        <div class="navbar-nav ms-auto">
+          <li class="nav-item">
+            <a class="nav-link text-white mx-4" aria-current="page" href="#"
+              >About</a
+            >
+          </li>
+          <a class="nav-link text-white mx-4" href="#">Work</a>
+          <a class="nav-link text-white mx-4" href="#">Contact</a>
+        </div>
+      </div>
+    </nav>
+
+    <section id="welcome-section" class="text-center">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+      <div class="container">
+        <h1 class="text-center">These are some of my projects</h1>
+        <div class="line mx-auto mb-5"></div>
+        <div class="row">
+          <div class="col-lg-4 col-sm-6 mb-4">
+            <div class="card">
+              <div style="background-color: #eeeeee" class="card-header my-0">
+                <a class="text-decoration-none" href=""
+                  ><h3 class="my-0 text-center text-dark">
+                    Dr. Normal Borlaug
+                  </h3></a
+                >
+              </div>
+
+              <img
+                class="img-fluid m-0"
+                src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+                alt="" />
+              <button style="background-color: #303841" class="btn">
+                Tribute Page
+              </button>
```

Comment:
Please consider using external css and avoid in-line styling 😕 since adding in-line css styling make your code complicated 😜